packages/content/src/prompts.ts

```diff
@@ -0,0 +1,159 @@
+import { FnDocumentation } from "@quri/squiggle-lang";
+
+import {
+  allDocs,
+  allMetas,
+  allRawApiDocs,
+} from "../.content-collections/generated/index.js";
+import { generateApiFunctionSection } from "./apiUtils.js";
+
+function moduleItemToCompressedFormat({
+  name,
+  description,
+  nameSpace,
+  signatures,
+  shorthand,
+  examples,
+}: FnDocumentation): string {
+  // Function signature line
+  const sigLine = `${nameSpace ? nameSpace + "." : ""}${name}${shorthand ? " " + shorthand.symbol : ""}: ${signatures.join(", ")}`;
+
+  // Description
+  const descLine = description ? `\n${description}` : "";
+
+  // Examples
+  let exampleLines = "";
+  if (examples && Array.isArray(examples) && examples.length > 0) {
+    exampleLines = "\n" + examples.map((e) => e.text).join("\n");
+  }
+
+  return `${sigLine}${descLine}${exampleLines}\n`;
+}
+
+function convertSquiggleEditorTags(input: string): string {
+  // Replace opening tags and everything up to the closing />
+  let result = input.replace(
+    /<SquiggleEditor[\s\S]*?defaultCode=\{`([\s\S]*?)`\}\s*\/>/g,
+    (match, codeContent) => {
+      return "```squiggle\n" + codeContent.trim() + "\n```";
+    }
+  );
+
+  return result;
+}
+
+function allDocumentationItems() {
+  return allRawApiDocs
+    .map((doc) => {
+      return (
+        `# Builtin module: ${doc.title}\n\n` +
+        doc.description +
+        "\n\n" +
+        doc.content +
+        "\n\n" +
+        generateApiFunctionSection(doc, moduleItemToCompressedFormat)
+      );
+    })
+    .join("\n\n\n");
+}
+
+function getDocsInDirectory(directory: string) {
+  const docs = allDocs.filter((doc) => doc._meta.directory === directory);
+
+  // Sort docs by the order of the pages in the Guides metadata.
+  // This matches Fumadocs behavior, which we assume is the most natural ordering both for humans and LLMs.
```

Comment:
I'm not sure how important this is, previously the pages weren't sorted, but it wasn't very hard to implement.